rds/outputs.tf

```diff
@@ -0,0 +1,29 @@
+output "password" {
+  sensitive = true
+  value     = random_password.password.result
+}
```

Comment:
admin/root password

Reply:
There is only one password that can be. 
Do you want me to name the variable as `root_username` and `root_password`?

Reply:
yeah I think the fact that it should not really be used by a web app needs to be signified...